torchbenchmark/util/torch_nightly.py

```diff
@@ -64,32 +67,40 @@ def get_nightly_wheel_urls(packages:list, date:date,
     return rc
 
 def get_nightly_wheels_in_range(packages:list, start_date:date, end_date:date,
-                                py_version='cp37', platform_version='linux_x86_64'):
+                                py_version='cp37', platform_version='linux_x86_64', reverse=False):
     rc = []
     curr_date = start_date
-    while curr_date < end_date:
+    while curr_date <= end_date:
         curr_wheels = get_nightly_wheel_urls(packages, curr_date,
                                              py_version=py_version,
                                              platform_version=platform_version)
         if curr_wheels is not None:
             rc.append(curr_wheels)
         curr_date += timedelta(days=1)
-
+    if reverse:
+        rc.reverse()
     return rc
 
 def get_n_prior_nightly_wheels(packages:list, n:int,
-                               py_version='cp37', platform_version='linux_x86_64'):
+                               py_version='cp37', platform_version='linux_x86_64', reverse=False):
     end_date = date.today()
     start_date = end_date - timedelta(days=n)
     return get_nightly_wheels_in_range(packages, start_date, end_date,
-                                       py_version=py_version, platform_version=platform_version)
-
+                                       py_version=py_version, platform_version=platform_version, reverse=reverse)
 
 if __name__ == "__main__":
-    from tabulate import tabulate
-    # print(tabulate(get_n_prior_nightly_wheels(['torch', 'torchvision', 'torchtext'], 200)))
-    wheels = get_n_prior_nightly_wheels(['torch', 'torchvision', 'torchtext'], 1)
+    parser = argparse.ArgumentParser(description="Find recent pytorch nightly builds")
+    parser.add_argument("--pyver", type=str, default="cp37", help="PyTorch Python version")
+    parser.add_argument("--platform", type=str, default="linux_x86_64", help="PyTorch platform")
+    parser.add_argument("--priordays", type=int, default=0, help="Number of days")
+    parser.add_argument("--reverse", action="store_true", help="Return reversed result")
+    parser.add_argument("--packages", required=True, type=str, nargs="+", help="List of package names")
```

Comment:
it looks like you forgot to plumb args.packages into the function call below?

Reply:
Right, thanks!